docs/general/ddl/analyzers.rst

```diff
@@ -110,6 +110,17 @@ This is the same as the `standard-analyzer`_ analyzer.
 
 Uses the :ref:`lowercase-tokenizer` tokenizer.
 
+.. _plain-analyzer:
+
+``plain``
+----------
+
+``type='plain'``
+
+This is the same as the `keyword`_ analyzer.
```

Comment:
can you rephrase these three lines as a single paragraph?

something like:

> The ``plain`` analyzer is an alias for the `keyword`_ analyzer and cannot be extended. You must extend the `keyword`_ analyzer instead.

Reply:
👍